app/services/application-menu.ts

```diff
@@ -31,7 +33,27 @@ export class ApplicationMenuService extends Service {
   private buildMenu(): electron.Menu {
     // TODO: i18n
     return electron.remote.Menu.buildFromTemplate([
-      { role: 'appMenu' },
+      {
+        label: 'Streamlabs OBS',
+        submenu: [
+          { role: 'about' },
+          {
+            label: 'Preferences…',
+            accelerator: 'Command+,',
+            click: () => {
+              this.settingsService.showSettings();
```

Comment:
`this.isEditorFocused()` shouldn't be needed here, since there won't be any issues if someone accesses settings will navigated to a different page.  I think it would be a good idea to add the `if (this.appService.state.loading) return;` check though.  We force all child windows closed while in the loading state, so we should let people open up settings while in the loading state either.